tests/integration/components/frost-bunsen-form/renderers/select/options/ember-data-view-query-test.js

```diff
@@ -452,13 +452,13 @@ describe('Integration: Component / frost-bunsen-form / renderer / select Ember D
                   options: {
                     labelAttribute: 'label',
                     modelType: 'node',
+                    placeholder: 'Foo bar',
```

Comment:
This wasn't strictly necessary, I just wanted to make sure it worked here before trying it in the `multi-select`